ReleaseTooling/Sources/ZipBuilder/main.swift

```diff
@@ -105,14 +104,23 @@ struct ZipBuilderTool: ParsableCommand {
           help: ArgumentHelp("The minimum supported iOS version."))
   var minimumIOSVersion: String
 
-  /// The list of architectures to build for.
+  /// The minimum macOS Version to build for.
+  @Option(default: "10.12",
+          help: ArgumentHelp("The minimum supported macOS version."))
+  var minimumMACOSVersion: String
```

Comment:
After looking at it further, I think this and the others should be renamed to `minimumMacOSVersion`

Reply:
Done.